sdk/eventhub/Azure.Messaging.EventHubs/tests/Processor/EventProcessorLiveTests.cs

```diff
@@ -45,29 +48,29 @@ await using (var client = new EventHubClient(connectionString))
                 {
                     var initializeCalls = new ConcurrentDictionary<string, int>();
 
-                    // Create the event processor hub to manage our event processors.
+                    // Create the event processor manager to manage our event processors.
```

Comment:
Some of these scenarios feel like we should be able to mock them.    For example, I'm not sure why we would need to use Azure to make sure that the calling start calls the partition's initialize method.   That should be something we can do in isolation.

Not a priority for preview 3, but let's make sure that we have an item to revisit the tests and review them for preview 4.

Reply:
Agreed. I'll open (one more) issue. I'm glad we'll be getting a fourth preview.